templates/ghost/index.ts

```diff
@@ -37,7 +37,7 @@ export function generate(input: Input): Output {
   });
 
   services.push({
-    type: "mariadb",
```

Comment:
why did you switch from mariadb?

Reply:
Start from Ghost 5.0, MariaDB is not an officially supported database for Ghost.

https://github.com/TryGhost/Ghost/issues/14446